### DISCUSSION TITLE
Support for layout/bounding box changes

### DISCUSSION THREAD

**Discussion by fantaandcode** [fenix-hub/godot-engine.easy-charts#96] [Ideas]
I'm in a circumstance where I want to change the bounding boxes of my charts and I see that there are Chart properties, but they're a little limited. I'm making a simple tracking line chart for the price of various goods.
![image](https://github.com/fenix-hub/godot-engine.easy-charts/assets/1066456/1eef1fca-78c5-48ac-85cb-dee563b8853e)
I want to change the graph to only show the red boxes. Is there a way to do that?

**Comment by fenix-hub** [comment]
Hi @fantaandcode 
You can try playing with [these properties](https://github.com/fenix-hub/godot-engine.easy-charts/blob/5c8f22c19d8ff8e0e6c94af0e7703da7e9fdc829/addons/easy_charts/utilities/classes/plotting/chart_properties.gd#L21-L33) in the ChartProperties class.
You can set everything to false and then try to switch on what you need.
Take a look at [this](addons/easy_charts/examples/line_chart/Control.gd) example to manipulate and pass the ChartProperties object.
Let me know if this solves your case.

**Comment by fantaandcode** [comment]
I've used those and they're not nearly as granular as I'd like. Is there a way to hack it together?

**Reply by fenix-hub** [reply]
@fantaandcode 
if this is what you want to achieve
![image](https://github.com/user-attachments/assets/8efef2cc-5b6b-479b-a72d-9e7ec5d9db77)

you can just tweak the attributes of ChartProperties class as I mentioned:
```gdscript
    var cp: ChartProperties = ChartProperties.new()
    cp.colors.frame = Color.TRANSPARENT
    cp.colors.background = Color.TRANSPARENT
    cp.colors.grid = Color.TRANSPARENT
    cp.colors.ticks = Color.TRANSPARENT
    cp.colors.text = Color.TRANSPARENT
    cp.colors.bounding_box = Color.RED
    cp.draw_bounding_box = true
```
